mesonpy/__init__.py

```diff
@@ -291,6 +291,8 @@ def entrypoints_txt(self) -> bytes:
     @property
     def _debian_python(self) -> bool:
         """Check if we are running on Debian-patched Python."""
+        if sys.version_info >= (3, 10):
```

Comment:
I think the right version threshold is 3.10.3, see also https://github.com/mesonbuild/meson/pull/11133